ui/src/time_picker.tsx

```diff
@@ -79,8 +79,11 @@ const
     toolbarLabel: { maxWidth: '70%' }
   }),
   popoverProps: Partial<PopperProps> | undefined = {
+    container: () => document.querySelector(".ms-Panel") || document.body,
```

Comment:
This solution is fine, but I am wondering whether we couldn't use HTML parent of `XTimePicker` component for all cases.

Reply:
We could try this.

Reply:
Go ahead and let me know, but first try to understand why `onMouseDown` prevention is not enough since it's suspicious.

Reply:
Just a thought: maybe the pointerEvents from the next comment prevented the `onMouseDown` from firing.